uelc/templates/admin/login.html

```diff
@@ -15,7 +15,7 @@
 <p class="errornote">{{ error_message }}</p>
 {% endif %}
 <div id="content-main">
-<form method="get" action="{{ CAS_BASE }}cas/login">
+<form method="get" action="{{ CAS_BASE }}cas/login">{% csrf_token %}
```

Comment:
The `get` forms actually don't need the `csrf_token` -- it just adds some unnecessary stuff to the URL, so these can be removed.